db/migrate/20230401123456_add_missing_columns_to_text_layers.rb

```diff
@@ -0,0 +1,15 @@
+class AddMissingColumnsToTextLayers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :text_layers, :font_family, :string unless column_exists?(:text_layers, :font_family)
+    add_column :text_layers, :font_style, :string unless column_exists?(:text_layers, :font_style)
+    add_column :text_layers, :font_size, :integer unless column_exists?(:text_layers, :font_size)
+    add_column :text_layers, :line_height, :float unless column_exists?(:text_layers, :line_height)
+    add_column :text_layers, :letter_spacing, :float unless column_exists?(:text_layers, :letter_spacing)
```

Comment:
The migration file AddMissingColumnsToTextLayers contains a column named 'text_transformation', which seems to be a duplicate of the 'text_transform' column. This could lead to confusion and data inconsistency. Please verify if 'text_transformation' is necessary or if it was added by mistake.